### DISCUSSION TITLE
Daily rebuild of data

### DISCUSSION THREAD

**Discussion by retrodans** [11ty/eleventy#2907] [Q&A]
For an internal application we are using `eleventy-fetch` which is working perfectly.  The only issue we have is that the data source changes on a daily basis (at around midnight), so it would be good if the 11ty app updated its build at about 1am to ensure it is all inline for the new day.

What is the best way to achieve this?  In my head, I have 2 options:

1. Don't use `eleventy-fetch` for this, instead write some bespoke js to make the requests
2. Have `cron` run the build on a daily basis, and exclude these build files from the git repo using `.gitignore`

Out of the 2, the cron solutions feels like my preference, but I wonder if there is a better solution people may already be using.

Thanks for the help,
Dan

**Comment by insanity54** [comment]
I'm doing nightly builds on [one of my eleventy apps](https://github.com/insanity54/bricktubers) by using a [cron timer in my github actions](https://github.com/insanity54/bricktubers/blob/main/.github/workflows/nightly.yml) to trigger a [netlify build](https://docs.netlify.com/configure-builds/build-hooks/).
